packages/commons/src/auth/authData.ts

```diff
@@ -0,0 +1,23 @@
+import { z } from "zod";
+
+export const AuthToken = z.object({
+  purposeId: z.string().uuid(),
+});
+export type AuthToken = z.infer<typeof AuthToken>;
+
+export const RequesterAuthData = z.object({
+  purposeId: z.string().uuid().default(""),
```

Comment:
`purposeId` is defined as `uuid` but the default value does not satisfy the spec.
Is it necessary to have a default value?